networkd/networkd.ml

```diff
@@ -64,6 +64,9 @@ let options = [
 	"dracut-cmd-path", Arg.Set_string Network_utils.dracut, (fun () -> !Network_utils.dracut), "Path to the Unix command dracut";
 	"dracut-timeout", Arg.Set_float Network_utils.dracut_timeout, (fun () -> string_of_float !Network_utils.dracut_timeout), "Default value for the dracut command timeout";
 	"modinfo-cmd-path", Arg.Set_string Network_utils.modinfo, (fun () -> !Network_utils.modinfo), "Path to the Unix command modinfo";
+	"json-rpc-max-len", Arg.Set_int Jsonrpc_client.json_rpc_max_len, (fun () -> string_of_int !Jsonrpc_client.json_rpc_max_len), "Maximum buffer size for Json RPC response";
+	"json-rpc-read-timeout", Arg.Set_string Jsonrpc_client.json_rpc_read_timeout, (fun () -> !Jsonrpc_client.json_rpc_read_timeout), "JSON RPC response read timeout value in ns";
```

Comment:
What you get from user is always `string`, here you can convert the string value to `int64`

Reply:
Arg does not support int64. And I am not sure the impact vs risks introducing expansion to support int64 here.

Reply:
Once the configuration unit changes to `ms`, int is large enough to store the value. Just convert from int to int64 while using, it costs less than convert from string to int64.

Reply:
the mtime/mtime_clock uses int64, if we do int/ms, then that means extra conversion. Why are you obsessed with just a fewer zeros?

Reply:
I think the best reason to avoid the manual conversion from string, is that if we set the argument as an int we will fail to run immediately when we load the variables. 

I don't think converting int to int64 or int64 from string later will be such a burden, we are doing this in a IO bound function, the IO will take the most of the time anyways.

Reply:
If the ref were of type `int64`, then I think you can simply replace

`Arg.Set_int Jsonrpc_client.json_rpc_max_len`

by

`Arg.Int (fun x -> Jsonrpc_client.json_rpc_max_len := Int64.of_int x)` to make it work.